certbot/CHANGELOG.md

```diff
@@ -509,7 +509,7 @@ More details about these changes can be found on our GitHub repo.
 * `--dry-run` now requests fresh authorizations every time, fixing the issue
   where it was prone to falsely reporting success.
 * Updated certbot-dns-google to depend on newer versions of
-  google-api-python-client and oauth2client.
+  google-api-python-client and google-auth.
```

Comment:
I don't know this was an accidental search-and-replace, but past CHANGELOG entries should not change.

You could add a comment to the top of the CHANGELOG file indicating that `certbot-dns-google` now depends on `google-auth` rather than `oauth2client`.